app/view/templates/pages/comparison/show.html

```diff
@@ -18,10 +18,18 @@ <h1>
         the checkbox to add them to the chart and compare values.
       </p>
 
-      <div style="float: right;">
-        {{ post_button("Clear collected comparisons", url_for("comparison_clear_action")) }}
-      </div>
-      <div style="clear: both;"></div>
+      {% if data_source == 'session': %}
+        <div style="float: right;">
+          {{ post_button("Clear collected comparisons", url_for("comparison_clear_action")) }}
+        </div>
+        <div style="clear: both;"></div>
+      {% else: %}
+        <p class="notice-message">
+          Currently showing a comparison encoded in the URL. Click on the
```

Comment:
I suggest you remove the `encoded in the URL` -- it would probably confuse most of the users. 
If I get this right, this is like a "switch" between the comparison table and the plot -- so I would just give messages for that, e.g. "Click 'compare' to see your selected experiments"